### DISCUSSION TITLE
Object Pooling causing memory leak

### DISCUSSION THREAD

**Discussion by shakir103** [FirstGearGames/FishNet#556] [Q&A - DO NOT USE - ASK ON DISCORD]
Hi, I am using fish-net for my game. I am using object pooling in game when Client game disconnect  from server . network objects gets added in cache and also get destroyed on scene unload on server which is causing memory leak how I can avoid that happening

**Comment by FirstGearGames** [comment]
This should have been filed as a bug report.  Will review but closing this.

**Comment by FirstGearGames** [comment]
This has been resolved in 4.0.9 and 3.11.15.
